src/fiber/network.rs

```diff
@@ -3395,50 +3362,57 @@ where
         match command {
             // Need to handle the force shutdown command specially because the ChannelActor may not exist when remote peer is disconnected.
             ChannelCommand::Shutdown(shutdown, rpc_reply) if shutdown.force => {
-                match self.store.get_channel_actor_state(&channel_id) {
-                    Some(mut state) => {
-                        match state.state {
-                            ChannelState::ChannelReady() => {
-                                debug!("Handling force shutdown command in ChannelReady state");
-                            }
-                            ChannelState::ShuttingDown(flags) => {
-                                debug!("Handling force shutdown command in ShuttingDown state, flags: {:?}", &flags);
-                            }
-                            _ => {
-                                let error = Error::ChannelError(
-                                    ProcessingChannelError::InvalidState(format!(
-                                        "Handling force shutdown command invalid state {:?}",
-                                        &state.state
-                                    )),
-                                );
-
-                                let _ = rpc_reply.send(Err(error.to_string()));
-                                return Err(error);
-                            }
-                        };
+                if let Some(actor) = self.channels.get(&channel_id) {
+                    actor.send_message(ChannelActorMessage::Command(ChannelCommand::Shutdown(
+                        shutdown, rpc_reply,
+                    )))?;
+                    Ok(())
+                } else {
+                    match self.store.get_channel_actor_state(&channel_id) {
+                        Some(mut state) => {
```

Comment:
I guess our test will all go to above branch, in which scenario this branch will cover,maybe when restarting a killed node?